### ISSUE TITLE
"Cannot GET" error

### ISSUE THREAD

**Issue by notapas** [Nubisa/jxdocs#161]
Hi, I use JXCore Plesk for developing an application on an existing website.
My problem is that all the website is listening on port and visiting other pages of other applications on the same domain I get the error "Cannot GET <path>".
I don't have this problem using NodeJS on my local webserver.
Is there a way to limit JXCore instance only on certain pages?
Thanks

**Comment by ktrzeciaknubisa** [comment]
Not really. From the moment when you enable JXcore support for a domain, all http/https domain calls are redirected to JXcore app. However, there you can try there to implement your redirection logic.

Also, I'm not sure from where the "Cannot GET" error comes from.

**Comment by notapas** [comment]
Thank you. I will try also to install JXCore on local webserver. But if I try to enable a second level domain on the same server and install there only the jxcore app? 
Otherwise, is it possible tu use nodejs on plesk without jxcore? 
Thanks

**Comment by ktrzeciaknubisa** [comment]
Yes, you could use JXcore only on second level domain/domains and not on main one.

As for Node.JS - JXcore extension for Plesk suports only JXcore engine. All apps are executed against jx executable which adds certain features (including security) introduced only by JXcore.

**Comment by obastemur** [comment]
Perhaps you receive Cannot Get because of NGINX proxy not JXcore. If you provide some structural details on your website, we may help

**Comment by notapas** [comment]
Ok, the problem was NGINX proxy. In fact trying to disable it no longer gives me the error. 
Now I have a test server, CentOS Linux 7.0.1406 with panel Parallels Plesk v12.0.18
Another thing, in which folder npm modules on plesk are installed? I need it for socket.io connection and I have only the jxcore path usr/local/psa/var/modules/jxcore-support/jx_deb64/jx/
Thanks

**Comment by ktrzeciaknubisa** [comment]
Npm modules are installed into */usr/local/psa/var/modules/jxcore-support/native_modules/*. Please note, that calling `require('module_name')` searches also there, so there is no need to provide full path.

**Comment by notapas** [comment]
No, I'm trying to change the path of socket connection because it does not work:
var socket = io.connect( 'http://my_website_address.it:10500')

**Comment by ktrzeciaknubisa** [comment]
If you call it from the same machine, maybe the host 'my_website_address.it' is unknown (e.g. not defined in */etc/hosts/*, so 'http://127.0.0.1:10500' might work instead.

Also, if `10500` is a port value of domain's application assigned by JXcore extension - I would not recommend to use it directly. It may change in several circumstances (e.g. changing port range on **JXcore Configuration** tab). Nginx is supposed to handle this when receiving a request for 'http://my_website_address.it'.

Anyway disabling Nginx is not probably a good idea.

**Comment by notapas** [comment]
I don't want to disable Nginx for production, I did this in order to understand the problem. In fact i'm waiting for a response from Plesk assistance.
Yes, 10500 is a port assigned by jxcore. Is there a way to use another port instead of the one indicated by the jxcore configuration?
But I had tried to connect with 127.0.0.1 or localhost because the call is from the same machine/domain but it gave me a connection refused error (net::ERR_CONNECTION_REFUSED - on chrome browser).
I can not understand if I can use jxcore to realize my application..
Thanks

**Comment by ktrzeciaknubisa** [comment]
> Is there a way to use another port instead of the one indicated by the jxcore configuration?

Only if you will run jx application outside of JXcore extension.

> I can not understand if I can use jxcore to realize my application..

Also we don't have a full image of what your trying to do. As @obastemur mentioned: "If you provide some structural details on your website, we may help".

**Comment by obastemur** [comment]
Actually he can change the ports on JXcore Plesk extension
